stable/database/templates/service-clusterip.yaml

```diff
@@ -42,6 +42,7 @@ metadata:
     release: {{ .Release.Name | quote }}
   name: {{ template "database.fullname" . }}-{{ default .Values.admin.serviceSuffix.clusterip .Values.database.serviceSuffix.clusterip }}
 spec:
+  publishNotReadyAddresses: true
```

Comment:
In the example I experienced, with a multi-cluster deployment, both deployments are primary -  a secondary was not used.  Don't we need to add it to primary too?

Reply:
If I'm not mistaken, in your test you have used external access using NLBs. This will utilize the "xxx-balancer" services for the TEs, which is now configured with `publishNotReadyAddresses=true` (see _service.yaml_). I also changed the TE group `ClusterIP` services because they are used when external access using `Ingress` is configured. 

The `demo-clusterip` will most probably be used only for `direct` TE access. (i.e. not a TE returned by the admin) so I think that we shouldn't publish unready endpoints.